src/pages/komodo-defi-framework/api/v20/utils/change_mnemonic_password/index.mdx

```diff
@@ -0,0 +1,66 @@
+export const title = "Komodo DeFi Framework Method: Change Mnemonic Password";
+export const description = "The change_mnemonic_password method allows a user to update the password used to encrypt a mnemonic phrase in their local database.";
+
+# change\_mnemonic\_password
+
+The `change_mnemonic_password` method allows a user to update the password used to encrypt a mnemonic phrase in their local database.
+
+<Note>
+  While this will change the password used to encrypt the mnemonic phrase in the local database, it will not update the `wallet_password` value in the MM2.json config file.
+  This will need to be updated manually, otherise you will see a log error `Error initializing wallet: Error generating or decrypting mnemonic: Error decrypting mnemonic: HMAC error: MAC tag mismatch` on next launch.
+</Note>
+
+## Arguments
+
+| Structure         | Type   | Description                                                  |
+| ----------------- | ------ | ------------------------------------------------------------ |
+| current\_password | string | the currently set mnemonic password                          |
+| new\_password     | string | the new password to use for encrypting the mnemonic password |
```

Comment:
```suggestion
| new\_password     | string | the new password to use for encrypting mnemonic password |
```

Reply:
not important! I think we could even make these descriptions more personal e.g
 `your current mnemonic password`

Reply:
agree :)